src/kirby/components/tabs/tab-button/tab-button.component.ts

```diff
@@ -28,8 +29,13 @@ export class TabButtonComponent implements OnInit {
     this.removeWrapper();
   }
 
-  onClick(event: Event) {
+  onClick(event: Event, clickedSelected: boolean) {
```

Comment:
Consider better naming;
```suggestion
  onClick(event: Event, isSelected: boolean) {
```